### PR TITLE
More api implementations, better filesystem management, all default extensions available

### PR DESCRIPTION
**Pull request by CGNonofr**
AFAIK, we don't need any feature from monaco-editor anymore (I mean, on top of monaco-editor-core)

### CODE REVIEW

**Comment by kaisalmen** [comment]
@CGNonofr this looks like an TTF not loading issue:
image

**Comment by CGNonofr** [comment]
> @CGNonofr this looks like an TTF not loading issue: image

I've just tried and I have the exact same rendering in vscode, what about you?

**Comment by kaisalmen** [comment]
@CGNonofr my local vscode doesn't show the same behavior. It doesn't seem to use the standard makrdown support, although  is configured. I tried disabling all installed extensions, but no effect. Code completion does not work. Are you already on 1.80.1?

**Comment by CGNonofr** [comment]
Yes I am. What does it look like on your vscode?

**Comment by kaisalmen** [comment]
Like this:
image

and that is not correct. The extension is not working. Something weird is going on.

**Comment by CGNonofr** [comment]
Can you explain the issue? if I enable `wordBasedSuggestions`, I have the exact same behavior between here and vscode

**Comment by kaisalmen** [comment]
> Can you explain the issue? if I enable wordBasedSuggestions, I have the exact same behavior between here and vscode

Yes, but in local vscode I don't get suggestions in the math section, but that's no issue here

**Review by kaisalmen — DISMISSED** [review]
Great work. This will simplify/unify usage also for monaco-languageclient users The comment on the `@rollup` import is not blocking.

File: src/extension-tools.ts [hunk 1]
```diff
@@ -10,7 +10,11 @@ import { getParseErrorMessage } from 'vs/base/common/jsonErrorMessages'
 import { InputPluginOption, rollup } from 'rollup'
 import { addExtension } from '@rollup/pluginutils'
 import { IUserFriendlyViewsContainerDescriptor } from 'vs/workbench/api/browser/viewsExtensionPoint'
+import inject from '@rollup/plugin-inject'
```
[the remark this answers is not in the document]
Comment (CGNonofr) [reply src/extension-tools.ts:13]: We need it at that stage. But the usage of rollup for extension codes is completely removed in one of the next PRs

[the remark this answers is not in the document]
Comment (kaisalmen) [reply src/extension-tools.ts:13]: OK

**Pull request merged**